visidata/path.py

```diff
@@ -179,10 +179,6 @@ def name(self):
         'Filename without any extensions.  Not the same as pathlib.Path.'
         return self.base_stem
 
-    @lru_cache()
-    def stat(self, force=False):
```

Comment:
Removing this makes me nervous.  It should still work, as members that aren't defined explicitly fall back to self._path, but the @lru_cache will make this much faster for DirSheets which would otherwise stat the same file repeatedly.  How can pyflakes know this isn't used?

Reply:
Oh, my commit was unclear. I believe this `stat()` is redundant because `stat()` is redefined a couple of hundred lines later in the same class.
https://github.com/saulpw/visidata/blob/c9790737398636c15946180bb3836ebe27a777e4/visidata/path.py#L377

Reply:
Ah, you're totally right!